R/combine-mrs.R

```diff
@@ -0,0 +1,61 @@
+##' Combine multiple_response variables into a new multiple_response
+##'
+##' This function lets you combine multiple_response variables into a new multiple_response
+##' Arguments include at least two multiple_response variables as well as any metadata
+##' for the new variable that is being created. The argument useNAsubs
+##' is given the defult value FALSE. If it is made TRUE, any subvariables that
+##' are NA in the categorical variables will become subvariables.
+##' This function uses the data from the arrays in the order that they are
+##' given. If the first variable is NA for a given row of a subvariable,
+##' it will use the data from the second variable and so on.
+##' @param variable 1 to combine
+##' @param variable 2 to combine
+##' @param other desired attributes such as new variable alias, name, 
+##' description as well as any other variables you would like to combine
+##' @param a boolean describing whether subvariables that are NA in the
+##' original variables should be made into subvariables in the new
+##' variable
+##' @param a boolean describing whether 'selected' should be given priority
+##' if false, 'not selected' will be given priority
+##' @return a new variable with combined responses from the given variables
+##' @export
+## NOTE THAT THIS IS ONLY GOING TO WORK FOR VARIABLES WHERE THE CATEGORIES ARE SELECTED AND NOT SELECTED. IF THE CATEGORIES ARE MADE BY SPSS, THIS WON'T WORK.
+## I've thought of a fix but it'll take more time (and is not currently useful to me).
+
```

Comment:
Same, and move the two non-documentation comments above elsewhere (perhaps inside the function)